block/block_service.proto

```diff
@@ -71,23 +38,157 @@ import "stream/block.proto";
 import "stream/block_item.proto";
 
 /**
- * Publish a stream of blocks.
+ * Publish a stream of block items.
  *
- * Each item in the stream MUST contain one `BlockItem`.<br/>
+ * Each request in the stream MUST contain at least one `BlockItem`.<br/>
+ * Each request MAY contain more than one `BlockItem`.<br/>
+ * A single request MUST NOT contain `BlockItem`s from more than one block.<br/>
+ * Each request MAY contain a variable number of `BlockItem`s.<br/>
  * Each Block MUST begin with a single `BlockHeader` block item.<br/>
+ * If a `BlockHeader` is present in a request, it MUST be the first `BlockItem`
+ * in the `block_items` list.<br/>
  * The block node SHALL append each `BlockItem` to an internal structure
  * to reconstruct full blocks.<br/>
  * The block node MUST verify the block proof for each block before sending a
  * response message acknowledging that block.<br/>
- * Each Block MUST end with a single `BlockStateProof` block item.<br/>
- * The block node MUST verify the Block using the `BlockStateProof` to
- * ensure all data was received and processed correctly.<br/>
+ * Each Block MUST end with a single `BlockProof` block item.<br/>
```

Comment:
This is true because we will only buffer a block without a proof on the node, but won't send it to a block node, correct?

Reply:
The block data will be sent in batches to the block node as data is available.  What we require here is that the Block Proof _MUST_ arrive _before_ the Block Header for the _next_ block.
The Consensus Node should _not_ buffer any more than necessary for efficiency and correct ordering.

Reply:
Note, the protocol/handshake between a _publisher_ (e.g. a consensus node) and a _block node_ is defined [here](https://github.com/hashgraph/hedera-block-node/blob/main/docs/Protocol/publishBlockStream.md).